catalog/cloud-sql-auth-proxy/daemonset.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: cloud-sql-auth-proxy # kpt-set: ${daemonsetName}
+  namespace: default # kpt-set: ${namespace}
+  labels:
+    app.kubernetes.io/name: "cloud-sql-auth-proxy" # kpt-set: ${serviceName}
+    app.kubernetes.io/version: "1.30.0" # kpt-set: ${tag}
+    app.kubernetes.io/component: "database"
+    app.kubernetes.io/tier: "backend"
```

Comment:
Lets use set-labels to manage labels rather than setters.
https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/examples/set-labels-simple